plugins/lsp/language-servers/default.nix

```diff
@@ -597,6 +597,11 @@ with lib; let
       description = "typst-lsp for typst";
       package = pkgs.typst-lsp;
     }
+    {
+      name = "vala-language-server";
+      serverName = "vala_ls";
+      description = "vala-language-server for Vala";
+    }
```

Comment:
```suggestion
    {
      name = "vala-ls";
      description = "vala_ls for Vala";
      serverName = "vala_ls";
      package = pkgs.vala-language-server;
    }
```

Reply:
Done. I assume we're following the naming convention from [lspconfig's filetree](https://github.com/neovim/nvim-lspconfig/tree/master/lua/lspconfig/server_configurations) (just in kebab-case), right?